VotingApplication/VotingApplication.Web/Scripts/Controllers/ManagePollTypeController.js

```diff
@@ -44,13 +44,14 @@
         }
 
         function tentativeUpdatePoll() {
-            ManageService.getVotes($scope.poll.UUID)
-            .then(function (pollSummary) {
-                if (pollSummary.data && pollSummary.data.Winners && pollSummary.data.Winners.length > 0 &&
-                        ($scope.poll.PollType !== startingPollType ||
-                        ($scope.poll.PollType === 'Points' &&
-                            ($scope.poll.MaxPerVote !== startingMaxPerVote ||
-                            $scope.poll.MaxPoints !== startingMaxPoints)))) {
+            ManageService.getPollType($scope.manageId)
+            .then(function (data) {
+
+                if (data.PollHasVotes &&
```

Comment:
Reading this code again, it's a bit busy and complicated to sit in an `if` predicate. It would probably read better in a separate function named something like `hasPollChanged`.